lunar/distribution.yaml

```diff
@@ -1063,6 +1063,21 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: lunar-devel
     status: maintained
+  pid:
+    doc:
+      type: git
+      url: https://bitbucket.org/AndyZe/pid.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
I think you might be able to override that but I haven't seen any entry which is different. When implementing the check we will see if any existing entry uses a non-standard entry.